src/main/resources/db_de.properties

```diff
@@ -1,3 +1,4 @@
+BaseEntity.cannotSaveEntity = ${entity} wurde nicht gespeichert.
```

Comment:
```suggestion
BaseEntity.cannotSaveEntity = ${entity} kann nicht gespeichert werden:
```

Reply:
Actually, do we want to add punctuation at all? Would limit it to always be placed at the beginning.